kubernetes/README.md

```diff
@@ -0,0 +1,86 @@
+# Deploying OpenWhisk on Kubernetes
+
+[![Build Status](https://travis-ci.org/openwhisk/openwhisk-devtools.svg?branch=master)](https://travis-ci.org/openwhisk/openwhisk-devtools)
+
+The collection of scripts and Kubernets pods can be used to
+run OpenWhisk in a Kubernets cluster.
+
+#### Kubernetes Requirments
+
+* Kubernetes needs to be version 1.5+
+* Kubernetes pods can receive public addresses
+* Kubernetes has DNS deployed
+
+
+## Configure OpenWhisk on Kube
+
+Prep the Kube environment by setting up the OpenWhisk namespaces `openwhisk`
```

Comment:
Readme seems to begin very abruptly - i would follow a flow, and at least suggest a minimum capacity requirement on Kubes cluster, and then, if the images need to be built (and even if its optional), that section comes before, and finally we apply the yml files.